pymeasure/instruments/tcpowerconversion/tccxn.py

```diff
@@ -0,0 +1,392 @@
+    of two representing characters.
+    """
+    return tuple(int(b) for b in value.to_bytes(2, "big"))
+
+
+class PresetChannel(Channel):
+    values = values
+
+    load_capacity = Instrument.control(
+        "GU\x00{ch:c}\x00\x00", "TD{ch:c}\x01\x00%c",
+        """ percentage of full-scale value of the load capacity preset """,
+        preprocess_reply=lambda d: struct.unpack(">H", d[2:4]),
+        validator=strict_discrete_set,
+        values=range(101),
+    )
+
+    tune_capacity = Instrument.control(
+        "GU\x00{ch:c}\x00\x00", "TD{ch:c}\x02\x00%c",
+        """ percentage of full-scale value of the tune capacity preset """,
+        preprocess_reply=lambda d: struct.unpack(">H", d[4:6]),
+        validator=strict_discrete_set,
+        values=range(101),
+    )
+
+
+class CXN(Instrument):
+    """T&C Power Conversion AG Series Plasma Generator CXN
+    (also rebranded by AJA International Inc as 0113 GTC or 0313 GTC)
+
+    Connection to the device is made through an RS232 serial connection.
+    The communication settings are fixed in the device at 38400, stopbit one,
+    parity none. The device uses a command response system where every receipt
+    of a command is acknowledged by returning a '*'. A '?' is returned to
+    indicates the command was not recognized by the device.
+
+    A command messages always consists of the following bytes (B):
+    1B - header (always 'C'),
+    1B - address (ignored),
+    2B - command id,
+    2B - parameter 1,
+    2B - parameter,
+    2B - checksum
+
+    A response message always consists of:
+    1B - header (always 'R'),
+    1B - address of the device,
+    2B - length of the data package,
+    variable length data,
+    2B - checksum
+    response messages are received after the acknowledge byte.
+
+    :param adapter: pyvisa resource name of the instrument or adapter instance
+    :param string name: Name of the instrument.
+    :param kwargs: Any valid key-word argument for Instrument
+
+    Note: In order to enable setting any parameters one has to request control
+    and periodically (at least once per 2s) poll any value from the device.
+    Failure to do so will mean loss of control and the device will reset
+    certain parameters (setpoint, disable RF, ...)
+    """
+    # use predefined values method to allow reusing in the Channels
+    values = values
+    presets = Instrument.ChannelCreator(PresetChannel, (1, 2, 3, 4, 5, 6, 7, 8, 9),
+                                        prefix="preset_")
+
+    def __init__(self, adapter, name="T&C RF sputtering power supply", address=0, **kwargs):
+        self.address = address
+        kwargs.setdefault("asrl", dict(baud_rate=38400))
+        super().__init__(adapter,
+                         name,
+                         includeSCPI=False,
+                         write_termination="",
+                         read_termination="",
+                         **kwargs)
```

Comment:
I agree that it makes sense to hard code the `baud_rate`, but I propose to use `asrl=dict(baud_rate=38400)`. Although the hardware of this device offers only RS232 I for example use it via an RS232 ethernet converter and therefore access it via `"TCPIP:....:SOCKET"`. I think hard coding `baud_rate` as suggested would cause troubles (#334)

So I guess what I am saying is that I think the use of the [Single Interface](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#single-interface) part of the documentation should be discouraged, because problems will arise if a converter of the hardware interface is used.

Reply:
You can always override these defaults, if you supply an `Adapter` instance to the instrument.
That is the preferred way if you use a converter (similar to the PrologixAdapter).

Reply:
That's true, but it comes with the overhead that the user needs to also know (set him/herself) `read_termination` and `write_termination` which is all not necessary if one uses the [Multiple interfaces](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#multiple-interfaces) approach. Does the multiple interface approach come with any disadvantage?

Reply:
> Does the multiple interface approach come with any disadvantage?

Not really, only a little more direct (you have a direct assignment instead of the `asrl={'baud_rate': 38400}`). Either way is fine for this PR.

I thank you for pointing out the issue of converters. You may open an issue to discuss the discouragement of the single interface, in order to get more opinions and to implement the documentation change.

Reply:
I opened #815 to discuss this further.
I here implemented it now without setdefault.